napari/layers/shapes/_shapes_mouse_bindings.py

```diff
@@ -501,16 +758,25 @@ def _move(layer, coordinates):
                 )
             layer._rotate_box(angle, center=layer._fixed_vertex)
             layer.refresh()
+
     elif (
-        layer._mode in {Mode.DIRECT, Mode.ADD_PATH, Mode.ADD_POLYGON}
+        layer._mode
+        in [
+            Mode.DIRECT,
+            Mode.ADD_PATH,
+            Mode.ADD_POLYGON,
+            Mode.ADD_POLYGON_LASSO,
+        ]
         and vertex is not None
     ):
         layer._moving_coordinates = coordinates
         layer._is_moving = True
         index = layer._moving_value[0]
         shape_type = type(layer._data_view.shapes[index])
-        if shape_type != Ellipse:
-            # not DIRECT vertex moving of ellipse not implemented
+        if shape_type == Ellipse:
+            # DIRECT vertex moving of ellipse not implemented
+            pass
+        else:
```

Comment:
Maybe this change should be reverted? RAter plain TODO should be added.

Reply:
I agree. Not certain though why this shows up as being changed though. Perhaps due to pre-commit, but the same lines exist on main: [code section](https://github.com/napari/napari/blob/d8dd1d2f592618064d2b5ac0c770ac1eb4741e77/napari/layers/shapes/_shapes_mouse_bindings.py#LL234C1-L237C1).

Reply:
resolved with [7b0cb08](https://github.com/napari/napari/pull/5555/commits/7b0cb08405cec16ed2f4f8141ca9712869cc022f)